src/directives/AddrResolver.js

```diff
@@ -173,6 +173,7 @@ const AddrResolver = {
         } catch (e) {
           if (e.message.includes('Missing validator for currency: ')) {
             _this.isValidAddress = true;
+            _this.hexAddress = domain;
```

Comment:
`_this.hexAddress` should always be the hexAddress of the resolved name

Reply:
it wasn't getting set when it wasn't a domain name, and was unable to validate.  In which case the domain is the address passed into the resolver/ validator